Trident/offsetfinder.c

```diff
@@ -832,11 +858,13 @@ uint32_t find_task_for_pid(void) {
         case iPhone41_iOS932: return 0x2fcd80;
         case iPhone41_iOS933: return 0x2fcec0;
         case iPhone41_iOS934: return 0x2fcec0;
+        case iPhone51_iOS933: return 0x302fd4;
         case iPhone52_iOS920: return 0x2fbb8c;
         case iPhone52_iOS921: return 0x2fbc9c;
         case iPhone52_iOS930: return 0x302e3c;
         case iPhone52_iOS932: return 0x302df0;
         case iPhone52_iOS933: return 0x302fd4;
+        case iPhone52_iOS934: return 0x302fd4;// ? to be verified
```

Comment:
![captur56e](https://cloud.githubusercontent.com/assets/25037788/22946225/1567db34-f2f7-11e6-8dce-2bb18e306f72.PNG)
Sorry for no reply, i'm a little bit confused because I don't know where to search for this offset, on the picture, i've calculated 0x803026F4 - 0x80001000 = 0x3016F4 but I don't know if it's right,
Sorry for being idiot, i'm just starting out, and sorry fo rmy bad english too